assets/css/_media-queries.scss

```diff
@@ -8,7 +8,7 @@
       box-shadow: 0 0 0.5rem $shadow;
       flex: 3 3 0;
       height: auto;
-      min-width: 180px;
+      min-width: 300px;
       position: relative;
```

Comment:
That's cool. Just wondering.

On a reference iphone the menu (when in mobile allows some panning left/right within the side panel when touch+dragging.